testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -133,50 +139,157 @@ pub struct MinedNakamotoBlockEvent {
     pub signer_bitvec: String,
 }
 
+impl InnerStackerDBChannel {
+    pub fn new_miner_receiver() -> (Receiver<StackerDBChunksEvent>, Self) {
+        let (sender, recv) = channel();
```

Comment:
This channel can grow to unbounded size.  Is that something we should worry about?

Reply:
I don't think so -- the channel is garbage collected after every block production. It only contains the signer event messages, so if the total number of those for a single block production is creating memory pressure, there's other issues (and there wouldn't be an easy fix either: we wouldn't want the sender to stall on a full buffer, so we'd need to switch to polling the db for updates).